dtfs-central-api/src/types/portal/portal-session-user.d.ts

```diff
@@ -0,0 +1,14 @@
+import { Prettify } from '../types-helper';
+import { PortalUser } from '../db-models/portal-user';
+import { SessionBank } from '../session-bank';
+
+export type PortalSessionUser = Prettify<Omit<PortalUser,
```

Comment:
I think we should use `Pick` instead of `Omit` to define the type here, because:
* it makes it easier to see what properties are in the type, not requiring you to look at `PortalUser` and subtract these properties
* it's more reflective of how we're actually constructing the portal session user from the portal user
* it will prevent new properties magically getting added to this type if `PortalUser` is ever updated

I'm proposing the same change for `TfmSessionUser` as part of my work for FN-1051.

Reply:
If you change to `Pick` you might be able to remove `Prettify`? Not 100% sure on that though

Reply:
Seems like you still need `Prettify`, but `Pick` is much clearer here